src/composables/LetoModelizerApi.js

```diff
@@ -0,0 +1,39 @@
+import { api } from 'boot/axios';
+
+/**
+ * Retrieve (GET request) information about the current user.
+ * @param {string} sessionToken - The current user's session token.
+ * @returns {Promise<object>} Promise with the logged-in user information on success
+ * otherwise an error.
+ */
+export async function getUserInformation(sessionToken) {
+  const headers = {
+    headers: {
+      Accept: 'application/json',
+      'X-Parse-Application-Id': process.env.BACKEND_APP_ID,
+      'X-Parse-Session-Token': sessionToken,
+    },
+  };
+
+  return api.get('/api/users/me', headers);
+}
+
+/**
+ * Get all roles of a user.
+ * @param {string} userId - User id.
+ * @param {string} sessionToken - The current user's session token.
+ * @returns {Promise<object[]>} Return an array of roles. Role is a parseObject server, so see
```

Comment:
`Role is a parseObject server` what does it mean ?

Doc says `Role is a local representation of a role persisted to the Parse cloud.` 
Or `ParseObject is a local representation of data that can be saved and retrieved from the Parse cloud. `

Reply:
done